From 58eceb8a19cb893fc7e9abc3a725d8bb4761dec3 Mon Sep 17 00:00:00 2001
From: sauoq <sauoq6@gmail.com>
Date: Mon, 23 Mar 2015 09:34:21 -0400
Subject: [PATCH] Fixes libgd ver check in Makefile.PL and Build.PL

---
 Build.PL    | 6 ++++--
 Makefile.PL | 6 ++++--
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/Build.PL b/Build.PL
index ad31624..d3b5b8b 100644
--- a/Build.PL
+++ b/Build.PL
@@ -280,8 +280,10 @@ sub try_to_autoconfigure {
   ($$lib_gd_path = $libdir) =~ s!/[^/]+$!!;
   $$options      = $features;
 
-  my ($minor)    = $version =~ /^2\.\d+\.(\d+)$/;
-  $$options     .= " GD_UNCLOSEDPOLY GD_ANIMGIF GD_FTCIRCLE VERSION_33" if defined($minor) && $minor >= 33;
+  my ($minor, $patch)    = $version =~ /^2\.(\d+)\.(\d+)$/;
+  if ((defined($minor) && $minor > 0) or (defined($patch) && $patch >=33)) { 
+	  $$options     .= " GD_UNCLOSEDPOLY GD_ANIMGIF GD_FTCIRCLE VERSION_33";
+  }
 
   my @correct_inc = map {s/^-I// && $_} split /\s+/,$cflags;
   check_for_stray_headers($includedir,@correct_inc);
diff --git a/Makefile.PL b/Makefile.PL
index 80ce95a..2b7551f 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -299,8 +299,10 @@ sub try_to_autoconfigure {
   ($$lib_gd_path = $libdir) =~ s!/[^/]+$!!;
   $$options      = $features;
 
-  my ($minor)    = $version =~ /^2\.\d+\.(\d+)$/;
-  $$options     .= " GD_UNCLOSEDPOLY GD_ANIMGIF GD_FTCIRCLE VERSION_33" if defined($minor) && $minor >= 33;
+  my ($minor, $patch)    = $version =~ /^2\.(\d+)\.(\d+)$/;
+  if ((defined($minor) && $minor > 0) or (defined($patch) && $patch >= 33)) { 
+    $$options     .= " GD_UNCLOSEDPOLY GD_ANIMGIF GD_FTCIRCLE VERSION_33";
+  }
 
   my @correct_inc = map {s/^-I// && $_} split /\s+/,$cflags;
   check_for_stray_headers($includedir,@correct_inc);
