--- GD-1.19/demos/brushes.pl	Mon Sep 14 22:28:53 1998
+++ GD-1.19.new/demos/brushes.pl	Sat Oct  9 21:32:58 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use GD;
 
--- GD-1.19/demos/copies.pl	Mon Sep 14 22:28:53 1998
+++ GD-1.19.new/demos/copies.pl	Sat Oct  9 21:33:04 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use GD;
 
--- GD-1.19/demos/fills.pl	Mon Sep 14 22:28:53 1998
+++ GD-1.19.new/demos/fills.pl	Sat Oct  9 21:33:09 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use GD;
 
--- GD-1.19/demos/gd_example.cgi	Mon Sep 14 22:28:53 1998
+++ GD-1.19.new/demos/gd_example.cgi	Sat Oct  9 21:33:13 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use GD;
 
--- GD-1.19/demos/polys.pl	Mon Sep 14 22:28:53 1998
+++ GD-1.19.new/demos/polys.pl	Sat Oct  9 21:33:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use GD;
 
--- GD-1.19/demos/shapes.pl	Mon Sep 14 22:28:53 1998
+++ GD-1.19.new/demos/shapes.pl	Sat Oct  9 21:33:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use GD;
 
--- GD-1.19/demos/transform.pl	Mon Sep 14 22:28:53 1998
+++ GD-1.19.new/demos/transform.pl	Sat Oct  9 21:33:31 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use GD;
 
diff -urN GD-1.23/demos/truetype_test GD-1.23.paths/demos/truetype_test
--- GD-1.23/demos/truetype_test	Tue Aug 31 03:25:26 1999
+++ GD-1.23.paths/demos/truetype_test	Thu Feb 17 17:29:46 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use lib '../blib/lib','../blib/arch';
 use GD 1.20;
diff -urN GD-1.23/demos/ttf.pl GD-1.23.paths/demos/ttf.pl
--- GD-1.23/demos/ttf.pl	Tue Aug 31 03:15:27 1999
+++ GD-1.23.paths/demos/ttf.pl	Thu Feb 17 17:29:46 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use lib '../blib/lib','../blib/arch';
 use GD 1.20;
