--- Makefile.PL~	2008-04-21 16:12:25.000000000 +0200
+++ Makefile.PL	2008-12-01 18:32:10.637155295 +0100
@@ -232,6 +232,7 @@
 		    'Math::Trig' => 0,
 		    },
     'PM'        => { 'GD.pm' => '$(INST_LIBDIR)/GD.pm',
+	    	     'GD/Group.pm' => '$(INST_LIBDIR)/GD/Group.pm',
                      'GD/Polyline.pm' => '$(INST_LIBDIR)/GD/Polyline.pm',
                      'GD/Polygon.pm' => '$(INST_LIBDIR)/GD/Polygon.pm',
                      'GD/Simple.pm' => '$(INST_LIBDIR)/GD/Simple.pm',
